### DISCUSSION TITLE
Will the [insider] Tag option be released to the main branch

### DISCUSSION THREAD

**Discussion by rangulvers** [squidfunk/mkdocs-material#4204] [General]
In the insider version there is the option to split tags to different pages. Will this be pushed to the main branch?

**Comment by squidfunk** [comment]
Of course! As soon as the ['Chipotle' funding goal](https://squidfunk.github.io/mkdocs-material/insiders/#16000-chipotle) is hit, this feature will be available to all users 😊

**Reply by rangulvers** [reply]
that's a long way to go 😉

**Reply by squidfunk** [reply]
If you need it now it's only $10 away and you would support me in maintaining this and all other features 😉
